action.yml

```diff
@@ -19,4 +22,5 @@ runs:
   image: 'Dockerfile'
   args:
     - ${{ inputs.url }}
+    - ${{ inputs.followRedirects }}
```

Comment:
This is one bit that I've not been able to test locally. Specifically, if the user doesn't set `followRedirects` I'm assuming that the action will pass an empty arg here to docker run. If it doesn't then it might mean that `grade` is sent as the second arg which would then break the `entrypoint.sh` script which is now expecting there to always be 3 args.

Reply:
Yes, this will break the `entrypoint.sh`. Maybe we could just stick with `$2` for grade and add `followRedirects` as `$3`?

I did some testing and this seems to work fine in all scenarios. Adding the new parameter as `$3` and setting some defaults is a non breaking change so I think it feels good.

```bash
#!/bin/bash

GRADE=${2:-'B'}
FOLLOW_REDIRECTS=${3:-''}
```

Also, if we like we can add some instructions when calling without parameters.

```bash
#!/bin/bash

if [ $# == 0 ]; then
    echo "Usage: param1 [param2] [param3]"
    echo "* param1: URL to analyse"
    echo "* param2: The desired security grade of your HTTP response headers. Possible grades: A+, A, B, C, D, E, F. Defaults to B"
		echo "* param3: Follow redirects. Defaults to off, set to on to enable"
		exit 1
fi

GRADE=${2:-'B'}
FOLLOW_REDIRECTS=${3:-''}
```

Reply:
Yeah, that's nicer. I've updated the PR with your suggestions.